spring-cloud-huawei-config/src/main/java/com/huaweicloud/config/client/ServiceCombConfigClient.java

```diff
@@ -150,10 +157,14 @@ public ServiceCombConfigClient(String urls, HttpTransport httpTransport) {
    * @throws RemoteOperationException
    */
   public Map<String, String> loadFromKie(ServiceCombConfigProperties serviceCombConfigProperties,
-      String project)
+      String project, boolean isWatch)
       throws RemoteOperationException {
     Response response = null;
     Map<String, String> result = new HashMap<>();
+    //todo : wait for kie support discovery
```

Comment:
Maybe this code can added later, create a task for this. For each version of sch, must notice the versions of suppoted kie.

Reply:
I‘ve delete the code , but I save the interface for next pr.